src/rules/shared.ts

```diff
@@ -284,318 +307,77 @@ const replaceHeaderByName = (
   return { ...request, headers: replacedHeaders }
 }
 
-// @ts-expect-error we have commonjs set as module option
-if (import.meta.vitest) {
-  // @ts-expect-error we have commonjs set as module option
-  const { it, expect } = import.meta.vitest
-
-  const generateRequest = (content: string | null): Request => {
-    return {
-      method: 'POST',
-      url: 'http://test.k6.io/api/v1/foo',
-      headers: [
-        ['content-type', 'application/json'],
-        ['content-length', '1000'],
-      ],
-      cookies: [['security', 'none']],
-      query: [],
-      scheme: 'http',
-      host: 'localhost:3000',
-      content,
-      path: '/api/v1/foo',
-      timestampStart: 0,
-      timestampEnd: 0,
-      contentLength: 0,
-      httpVersion: '1.1',
-    }
+export function replaceAllBody(
```

Comment:
`replaceAllBody`, `replaceAllUrl`, and others are basically replacements for the old `replaceContent` `replaceURL`, etc. with a few minor changes (such as their names being more explicit about what the function does (although at this point I'd prefer all names to follow the "replace X in Y pattern", but that's a separate discussion), and the return type being `Request`).